VERSION

```diff
@@ -82,17 +82,17 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=20:2:0
-libmpi_cxx_so_version=20:1:0
-libmpi_mpifh_so_version=21:0:1
-libmpi_usempi_tkr_so_version=21:0:1
-libmpi_usempi_ignore_tkr_so_version=21:0:1
-libmpi_usempif08_so_version=21:0:1
-
-libopen_rte_so_version=21:0:1
-libopen_pal_so_version=22:0:2
+libmpi_so_version=20:3:0
+libmpi_cxx_so_version=20:2:0
+libmpi_mpifh_so_version=22:0:0
+libmpi_usempi_tkr_so_version=22:0:0
+libmpi_usempi_ignore_tkr_so_version=22:0:0
+libmpi_usempif08_so_version=22:0:0
```

Comment:
All the Fortran values need to be `22:0:2`, otherwise you are breaking backwards compatibility.

Remember: we know we made a backwards incompat change, but the type was wrong, so it's ok.  Hence, if I compile/link my Fortran application against OMPI v2.0.2, it should still run-time link against OMPI v2.0.3 shared libraries without recompiling.  Setting `a` to `0` breaks that possibility.